test/optimizations/remoteValueForwarding/serialization/globals.chpl

```diff
@@ -70,9 +71,13 @@ proc main() {
     stop("begin-on");
   }
 
-  for (msg, dat) in commStats.items() {
+  var keys = commStats.keys();
+  sort(keys);
+
+  for msg in keys {
```

Comment:
Just to be sure, this is a deprecation-related change, isn't it?

Reply:
Yeah, these tests shouldn't have been .futures. They never had a .bad file, so a couple of different changes over time meant that the tests kept failing when we could have fixed them.

Reply:
Oh, I didn't even realize that they were futures to begin with. Thanks!

It might be good to put a short sentence in the PR message for posterity.